### ISSUE TITLE
pyhtonbot !log doesnt work in private messages

### ISSUE THREAD

**Issue by carlbordum** [8Banana/nettirely#5]
pyhtonbot: Zaab1t: Uploading logs, this might take a second...
pyhtonbot: Zaab1t: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">

**Comment by PurpleMyst** [comment]
It seems that any command doesn't work in private messages. I'll look into this.

**Comment by PurpleMyst** [comment]
This should be fized @Zaab1t.

**Comment by carlbordum** [comment]
Hello, is the fix live on freenode?

This is the response I currently get:
`<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">`

**Comment by Akuli** [comment]
Doesn't seem to be fixed to me.

```
<Akuli> !log
<pyhtonbot> Akuli: Uploading logs, this might take a second...
<pyhtonbot> Akuli: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
```

**Comment by PurpleMyst** [comment]
Sadly I am unable to test this due to freenode spam. I would need to register another pyhtonbot account just for testing, seeing as the master branch is currently sacred and all freenode users have mode +R by default. I'll have to figure out how to mode -R myself.

**Comment by PurpleMyst** [comment]
This is fixed in c30d38dbd039f2f37f4dbb7db293382a6b6b7713. It was a pretty easy bug, just depended on changing the key when we recieve the PRIVMSG from the bot's nickname to the sender's.
